hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractMintHTSSuite.java

```diff
@@ -191,7 +191,7 @@ private HapiSpec transferNftAfterNestedMint() {
                                                                     .withStatus(SUCCESS)
                                                                     .withTotalSupply(1L)
                                                                     .withSerialNumbers(1L))
-                                                            .gas(3_837_920L)
+                                                            .gas(3_836_587L)
                                                             .amount(0L)
```

Comment:
And again here?  I have no real objection to finding these numbers by running the test and seeing what works and what fails, it's just unfortunate that that's the way it has to work.